lab_08/main.c

```diff
@@ -0,0 +1,116 @@
+#include <stdio.h>
+#include <time.h>
+#include <stdlib.h>
+
+#define SIZE 8
```

Comment:
Please remove it and fix the code.

Reply:
What do you mean by removing? Check please the new push, I've checked via valgrind, understand the issue (memory leak) and fixed it.